src/main/java/org/elasticsearch/action/admin/cluster/repositories/verify/TransportVerifyRepositoryAction.java

```diff
@@ -69,7 +69,7 @@ protected VerifyRepositoryResponse newResponse() {
 
     @Override
     protected ClusterBlockException checkBlock(VerifyRepositoryRequest request, ClusterState state) {
-        return state.blocks().indexBlockedException(ClusterBlockLevel.METADATA, "");
+        return state.blocks().indexBlockedException(ClusterBlockLevel.METADATA_WRITE, "");
```

Comment:
maybe I don't know what verify repo does, bu tit sounds like a read_only operation to me. Why METADATA_WRITE and not METADATA_READ?